package.json

```diff
@@ -6,6 +6,8 @@
   "types": "index.d.ts",
   "scripts": {
     "test": "standard | snazzy && tap test/*test.js && tsd",
+    "test:prepare": "node test/browser/helpers/runner-prepare.mjs",
```

Comment:
rename "test:prepare" to "pretest:browser" ?

Reply:
Unfortunately, script hooks have a limitation and that is that it is unable to forward arguments to the main script.

One solution might be to create another file that executes both: the prepare and the runner. At this point we will only have one script, it also seems to me a much cleaner solution. 

What do you think?

Reply:
Is it possible to include the `setup` script (let's use `setup` to identify pre* operations) in the tests, and check on there if the setup phase is done?